game.js

```diff
@@ -1,4 +1,5 @@
 function gameEntryPoint() {
+    document.getElementsByTagName("body").item(0).style.background = "var(--theme-primary)"
```

Comment:
Neni tohle taky nadbytečné?

Reply:
Netuším proč, ale bez tohoto řádku mně to nefungovalo a furt to dělalo bílý pozadí nezávisle na zapnutým módu.

Reply:
Jsme z toho zmatený proč to tak je, ale máš naprostou pravdu